src/main/kotlin/com/aoe/alloypricecalculator/web/AuthenticationController.kt

```diff
@@ -0,0 +1,30 @@
+package com.aoe.alloypricecalculator.web
```

Comment:
maybe rename to stick with the ddd naming convention
```suggestion
package com.aoe.alloypricecalculator.interface
```